software/receipt-wrangler.yml

```
@@ -0,0 +1,14 @@
+name: "Receipt Wrangler"
+website_url: "https://receiptwrangler.io"
+source_code_url: "https://github.com/Receipt-Wrangler"
+description: "An easy to use Receipt manager, powered by AI. Allows users to create receipts effortlesly and quickly, catagorize and more."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Go
+  - Python
+  - Docker
```

```suggestion
platforms:
  - Docker
```

Docker is the only supported installation method https://github.com/Receipt-Wrangler#step-1-set-up-docker-composeyaml